defaults/main.yml

```diff
@@ -41,3 +41,11 @@ es_debian_startup_timeout: 10
 
 # JVM custom parameters
 es_jvm_custom_parameters: ''
+
+# SSL/TLS parameters
```

Comment:
This new variables should have a description in the README file

Reply:
good catch, of course you're correct